src/operator/subgraph/mkldnn/mkldnn_conv.cc

```diff
@@ -373,6 +374,19 @@ void SgMKLDNNConvOperator::Forward(const OpContext &ctx,
       if (mkldnn_param.with_sum) {
         full_conv_param.sum_scale = output_scale / sum_in_scale;
       }
+      if (mkldnn_param.with_act &&
+          full_conv_param.act_param.alg == mkldnn::algorithm::eltwise_bounded_relu) {
+        if (mkldnn_param.with_sum) {
+          LOG(ERROR) << "mkldnn doesn't support conv + relu + sum fusion yet.";
```

Comment:
Is this a `warning`  and make the unfused flow can work too?

Reply:
This is unreachable code, because conv + relu + sum fusion is disabled in subgraph fusion pass. So Error is acceptable here, and shouldn't break unfused flow.